GithubBrowserSample/app/src/main/java/com/android/example/github/ui/repo/RepoFragment.kt

```diff
@@ -46,7 +46,7 @@ import javax.inject.Inject
 class RepoFragment : Fragment(), Injectable {
 
     @Inject
-    lateinit var viewModelFactory: ViewModelProvider.Factory
+    lateinit var viewModelFactory: GithubViewModelFactory<RepoViewModel>
```

Comment:
using view model delegates you do not need to use a viewModelFactory

Reply:
Not sure I follow your point. The code is already using the delegates
```kotlin
val repoViewModel: RepoViewModel by viewModels {
        viewModelFactory
    }
```
And the `viewModelFactory` is needed because `RepoViewModel` takes a `RepoRepository` in constructor. Without a factory, the delegate wouldn't know how to create the view model.